.cirrus.yml

```
@@ -311,6 +311,15 @@ netdata_task:
   env:
     PLUGIN_FILE: "netdata.json"
 
+nexus_task:
+  <<: *INSTALL_PLUGIN
+  only_if: "changesInclude('nexus-oss.json', '.cirrus/install_script.sh')"
+  matrix:
+    - freebsd_instance:
+        image_family: freebsd-12-1
```

Probably want to use `12-2` here
```suggestion
        image_family: freebsd-12-2
```

---

yeah, nice catch.